src/hooks/routing/useAutoRouterSupported.ts

```diff
@@ -0,0 +1,8 @@
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+
+import { AUTO_ROUTER_SUPPORTED_CHAINS } from './clientSideSmartOrderRouter'
```

Comment:
Change this to a helper function following the example in https://github.com/Uniswap/interface/pull/3915/commits/4c492e3b09b888444130e8491510f176d22c1cfe. You should do this in a separate PR, and it may be helpful to do this for all instances of `.includes` checks against lists.